addons/website/views/snippets/s_avatars.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template name="Avatars" id="s_avatars">
+    <div class="s_avatars d-flex flex-row flex-wrap"
+         style="--avatars-size: 3rem"
+        >
+        <div class="s_avatars_wrapper oe_unremovable d-flex">
+            <div
+                class="s_avatars_rules_collector d-contents rounded-circle oe_unremovable"
+                style="border: 3px solid var(--body-bg);"
+            >
+                <img src="/web/image/website.s_company_team_image_1"
+                     class="img o_avatar"
```

Comment:
Nitpick: I think that a `s_avatars_avatar` class could be added (this is a bit the convention in other snippets from what I saw) 
Also in `s_avatars.avatar` template and even in `s_avatars.more` so you can target them more easily since they share everything (like instead of `img.o_avatar, .s_avatars_more` in the SCSS, you could just do `.s_avatars_avatar`)  :slightly_smiling_face:

But this is an nitpick since they don't really have their own options, so as you prefer :slightly_smiling_face: